modeling-cmds/src/def_enum.rs

```diff
@@ -279,6 +279,21 @@ define_modeling_cmd_enum! {
             pub rotate_duplicates: bool,
         }
 
+        /// Create a helix using the input cylinder and other specified parameters.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ModelingCmdVariantEmpty)]
+        pub struct EntityMakeHelix {
+            /// ID of the cylinder.
+            pub cylinder_id: Uuid,
+            /// Number of revolutions.
+            pub revolutions: f64,
+            /// Start angle (in degrees).
+            pub start_angle: units::UnitAngle,
```

Comment:
This is just a unit (degrees or radians), it should be the `Angle` struct (which is an angle size and a unit the size is measured in, e.g. {40, degrees} or {2, radians})